privacyidea/static/components/directives/views/directive.policyconditions.html

```diff
@@ -0,0 +1,62 @@
+<div class="table-responsive">
+    <table class="table">
+       <tr ng-repeat="condition in policyConditions">
```

Comment:
Maybe we should add a table header to describe the entries in the columns in more detail, so that a user knows, what to put where.

Reply:
Good idea! I added table headings in 65ee04ee